internal/source/source.go

```diff
@@ -92,19 +92,16 @@ func scanToLine(fileset *token.FileSet, node ast.Node, lineNum int) ast.Node {
 
 func getCallExprArgs(fileset *token.FileSet, astFile ast.Node, line int) ([]ast.Expr, error) {
 	node, err := getNodeAtLine(fileset, astFile, line)
-	switch {
-	case err != nil:
+	if err != nil {
 		return nil, err
-	case node == nil:
-		return nil, fmt.Errorf("failed to find an expression")
 	}
 
 	debug("found node: %s", debugFormatNode{node})
 
 	visitor := &callExprVisitor{}
 	ast.Walk(visitor, node)
 	if visitor.expr == nil {
-		return nil, errors.New("failed to find call expression")
+		return nil, fmt.Errorf("failed to find an expression")
```

Comment:
There is no reason to use fmt.Errorf when your are not formatting


```suggestion
		return nil, errors.New("failed to find an expression")
```

Reply:
oh, good call; not sure why I changed this one!

oh; I see - I guess I copy/pasted it from the original one 🙈

Reply:
Fixed 👍